### PR TITLE
brush: update documentation section

### PR DESCRIPTION
**Pull request by rzerres**
Signed-off-by: Ralf Zerres <ralf.zerres@networkx.de>

# Context:
Update the documentation part of gradient implementation for brush.rs

Reference to a related issue in the repository.
@FloVanGH 

## Contribution checklist:
- [x] Add [documentation](https://doc.rust-lang.org/1.7.0/book/documentation.html) to all public structs, traits and functions.
- [ ] Add unit tests if possible
- [x] Describe the major change(s) in the CHANGELOG.MD
- [x] Run `cargo fmt` to make the formatting consistent across the codebase
- [x] Run `cargo clippy` to check with the linter

### CODE REVIEW

**Comment by sandmor** [comment]
I like some improvements but you vanish completely my extensive explanation of the range of the different parameters of the functions and also delete the extra color list, I think that things must be conserved, remove that only undermine the documentation.

File: crates/utils/src/brush.rs [hunk 1]
```diff
@@ -1,67 +1,76 @@
...
 ///
-/// B. Use a function, currently the unique available functions to interpret a color are `rgb`, `hsv`, `hsb`
-/// (which is the same as `hsv`), `hsl` and its respective alpha variants(rgba by example). So you can define
-/// a color as `hsl(197, 71%, 73%)` and you will get a pretty skyblue color. For `rgb` and `rgba` the
-/// range of the values are 0-255 but from all others the values are as follows: hsva(0.0-360.0, 0.0-1.0, 0.0-1.0, 0.0-1.0)
-/// but the % sign can change that, it can be added to individual parameters and change its range depending on the function,
-/// for `rgb` and `rgba` you can write an % after a parameter and it will be mapped from 0.0-100.0, or if the number that you write
-/// is between 0.0 and 1.0 you value will be mapped from that range. But if you function is not rgb or rgba, you can not use a
-/// percent sign in the first parameters but in the following parameters the sign will make OrbTk interpret it in the 0.0-100.0 range.
+/// Define the value with a symbol "#" followed by 6 letters or numbers.
```
Comment (sandmor) [inline crates/utils/src/brush.rs:10]: the lengths are 3, 6 and 4, 8 for non-alpha and alpha colors respectively. I prefer be clear and verbose

File: crates/utils/src/brush.rs [hunk 2]
```diff
@@ -1,67 +1,76 @@
...
+/// parameters.
+/// When appending the `%` sign to the range parameters of the `rgb` function
+/// call, the values are mapped to 0.0-100.0 (percent) or 0.0-1.0 (min/max).
+/// For all other keywords (`hsv`, `hsb`, `hsl`) you are not allowed to append
+/// the percent sign to the first parameter. If you append `%` to the following
+/// parameters, OrbTk will interpret the values in a range between `0.0-100.0`.
+///
+/// C. Use the `color name`
+///
+/// OrbTK maintains a color name map (crates/utils/colors.txt). It enables
```
Comment (sandmor) [inline crates/utils/src/brush.rs:36]: Yes you can do a link to the original but that forces the user to look at the source code and do an extra step, I don't think extensive documentation is a problem(I always use the index to navigate the docs like most people do)

**Review by sandmor — COMMENTED** [review]
In general, I like the improvements(except for the removal of the color list) but I think that verbose and well-explained documentation is better than a short although well written in that last I applaud you, you always can use the index to navigate, do you know?

**Comment by rzerres** [comment]
@sandmor I did rework the documentation as discussed last night. Let me know, if i got that right.
@FloVanGH  did send you a privat message on mattermost describing the ongoing discussion concerning centralized handling of color names in utils crate (const_colors.rs)

**Review by sandmor — COMMENTED** [review]
Great!, thank you for your work

**Comment by rzerres** [comment]
fine @sandmor,
can you please mark the PR as approved? Then it is up to @FLoVanGH to merge it in.

**Review by sandmor — APPROVED** [review]

**Pull request merged**